src/fidocadj/ADesktopIntegration.java

```diff
@@ -48,96 +49,107 @@ public class ADesktopIntegration implements AboutHandler, PreferencesHandler,
     }
 
     /** Check if some actions are made available by the operating system and if
-        it is the case, register them.
-        For example, MacOS usually inserts the About and Preferences menus in a
-        specific place in the "FidoCadJ" menu, that does not exist in other
-        platforms. This means that the OS needs to know what to call when one
-        of those two actions is selected. This is given to the operating system
-        by this routine.
-    */
+     * it is the case, register them.
```

Comment:
I tend not to use the * at the beginning of each Javadoc comment between /** and */, but this is not a big deal